nubilanding/src/App.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import './App.css';
+import Blog from "./components/blog/blog";
 
 function App() {
   return (
     <div className="App">
-
+    <Blog />
```

Comment:
No te olvides del pobre identado(?
```suggestion
      <Blog />
```